integration_tests/features/support/steps.js

```diff
@@ -54,6 +56,14 @@ Given(/I have a pruned node (.*) connected to node (.*)/, {timeout: 20*1000}, as
     await sleep(1000);
 });
 
+Given(/I have a lagging delayed node (.*) connected to node (.*) with blocks_behind_before_considered_lagging (\d+)/, {timeout: 20*1000}, async function (name, node, delay) {
+    const miner =  new BaseNodeProcess(name, { blocks_behind_before_considered_lagging: delay});
```

Comment:
Looking, just one comment.

Maybe my bad, but I do not understand how this can have any effect if you have not added the option to [`function mapEnvs(options) {` in `config.js`](https://github.com/SWvheerden/tari/blob/sw_Add_delay_lagging/integration_tests/helpers/config.js#L7) (or see [main repo](https://github.com/tari-project/tari/blob/development/integration_tests/helpers/config.js#L7)) ?